NCS.DSS.GDPRUpdateCustomerInformation.Tests/NCS.DSS.GDPRUtility.Tests.csproj

```diff
@@ -15,6 +15,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="FakeItEasy" Version="8.3.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.23.0" />
```

Comment:
Uninstall this package - it's not used

Reply:
it's required for the timer

Reply:
as above, it's required for the isolated worker model: https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide

Reply:
It's not required: 
![image](https://github.com/user-attachments/assets/4430ea0a-d143-47ba-bef1-7da591f25bbd)

This was deployed to AT without that package - please uninstall

Reply:
It's building for me today... how strange that it says it was required